nbproject/project.properties

```diff
@@ -41,7 +41,11 @@ j2ee.compile.on.save=true
 j2ee.copy.static.files.on.save=true
 j2ee.deploy.on.save=true
 j2ee.platform=1.7-web
+<<<<<<< HEAD

```

Comment:
Resolve this conflict. This will end up breaking the project.properties file. In fact, I could have sworn I added this to the .gitignore. This and ant-deploy.xml have to be filtered out.